src/Controls/tests/Core.UnitTests/ViewUnitTests.cs

```diff
@@ -662,10 +662,16 @@ public void ClearingGestureRecognizers()
 			var gestureRecognizer = new TapGestureRecognizer();
 
 			view.GestureRecognizers.Add(gestureRecognizer);
-			view.GestureRecognizers.Clear();
-
 
+#if !ANDROID
```

Comment:
These tests don't run on a device they just run as a netstandard library so the ANDROID definition areas will have no effect.